src/index.ts

```diff
@@ -23,16 +29,43 @@ const extension: JupyterFrontEndPlugin<void> = {
   id: PLUGIN_NAME,
   autoStart: true,
   activate: activateCellVim,
-  requires: [INotebookTracker, ICodeMirror, ISettingRegistry]
+  requires: [INotebookTracker, IEditorExtensionRegistry, ISettingRegistry]
 };
 
 async function activateCellVim(
   app: JupyterFrontEnd,
   tracker: INotebookTracker,
-  jlabCodeMirror: ICodeMirror,
+  editorExtensionRegistry: IEditorExtensionRegistry,
   settingRegistry: ISettingRegistry
 ): Promise<void> {
-  // await app.restored;
+  const theme = Prec.highest(
+    EditorView.theme({
+      '.cm-fat-cursor': {
+        position: 'absolute',
+        background: 'var(--jp-vim-cursor-color)',
+        border: 'none',
+        whiteSpace: 'pre'
+      },
+      '&:not(.cm-focused) .cm-fat-cursor': {
+        background: 'none',
+        outline: 'solid 1px var(--jp-vim-cursor-color)',
+        color: 'transparent !important'
```

Comment:
this is a nice addition

Reply:
Week, it overrides the upstream style because they changed colour to red/orange from the previous green and I thought the green one was better. I thought that instead of hardcoding it would be good to allow themes to override it.

Reply:
Is it this or something else that also seems to have fixed the issue with highlighting during search? That also seems resolved now :)